scene/main/viewport.cpp

```diff
@@ -37,7 +37,7 @@
 #include "core/templates/pair.h"
 #include "scene/2d/camera_2d.h"
 #include "scene/2d/collision_object_2d.h"
-#include "scene/2d/listener_2d.cpp"
+#include "scene/2d/listener_2d.h"
```

Comment:
This fixed a multiple definition error which I assume was hidden from static builds.

Reply:
got fixed in https://github.com/godotengine/godot/pull/52612